### PR TITLE
Force Update Error Format

### PR DESCRIPTION
**Pull request by Busta117**

### CODE REVIEW

**Comment by danielgomezrico** [comment]
@Busta117 we will have more than one error format? current format is:

```
{
  "errors": {
    "field1": "message1",
    "field2": "message2"
  }
}
```

File: backend/README.md [hunk 1]
```diff
@@ -33,3 +33,16 @@ All errors from backend should have in body:
...
 }
 ```
+* Foce Update:
+	- error have to include a message to show in the alert
+	- error have to include an URL for the app redirect
+
+error response will be in the following format:
+```
+{
+  "errors": {
```
Comment (danielgomezrico) [inline backend/README.md:43]: the content of `errors` is not multiple errors, it looks weird to me.

**Review by danielgomezrico — CHANGES_REQUESTED** [review]

File: backend/README.md (hunk 1 above)
Comment (edwaraco) [reply backend/README.md:43]: @caipivara Errors may content multiples attributes with errors. For this reason, they are wrapped in `errors`.

Comment (danielgomezrico) [reply backend/README.md:43]: @earayo but what I mean is that errors have two different formtats, one for errors where key is the reason of the error and message the message to show, and other for force update with `url` and `message` field.

**Review by orendon — APPROVED** [review]
I agree here, actually MedAnswers and Lenspire are following this convention already.

File: backend/README.md (hunk 1 above)
Comment (orendon) [reply backend/README.md:43]: The force update proposed by @Busta117 is ok, and it's being used now in all projects.

Let's continue the general error handling strategy on #18

**Review by danielgomezrico — APPROVED** [review]

**Pull request merged**